packages/storage-plus/src/prefix.rs

```diff
@@ -151,6 +170,43 @@ where
             range_with_prefix(store, &self.storage_prefix, min, max, order).map(|(k, _)| k);
         Box::new(mapped)
     }
+
+    pub fn range_de<'a>(
```

Comment:
There is no need for `range_de` as far as I can tell.

It is the same as `range`, right?
You just set `K = Vec<u8>` and this will work like the "legacy version"

Reply:
Exactly.